### DISCUSSION TITLE
Unable to install truffle

### DISCUSSION THREAD

**Discussion by kahiga254** [ConsenSys-archive/truffle#6225] [Installation]
I'm unable to install truffle on my computer I keep on getting the error 

npm ERR! code ERR_SOCKET_TIMEOUT
npm ERR! network Socket timeout
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

any solution to this will be highly appreciated.

**Comment by MadelineMurray** [comment]
hi @kahiga254  - Truffle is being sunset. For more information and migration options, click [here](https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat?utm_source=github&utm_medium=referral&utm_campaign=2023_Sep_truffle-sunset-2023_announcement_).
